src/reader.rs

```diff
@@ -1109,6 +1109,7 @@ where
                     }
                 }
                 Event::Comment(_) => {}
+                Event::Empty(_) => {} // Need o handle empty element to properly parse the tree
```

Comment:
Can we simplify this to

```rust
Event::Comment(_) | Event::Empty(_) => {}
```

And since we're doing this in other places I think we can drop the comment